services/httpd/handler.go

```diff
@@ -236,6 +238,15 @@ func (h *Handler) Open() {
 		h.Logger.Info("opened HTTP access log", zap.String("path", path))
 	}
 
+	for _, s := range h.Config.AccessLogStatusFilters {
+		sf, err := newStatusFilter(s)
+		if err != nil {
+			h.Logger.Warn("Dropping invalid access log status filter", zap.String("filter", s), zap.Error(err))
```

Comment:
It would be better to properly fail on an invalid access log filter. There are often a lot of logs flying by at startup and it would be easy to miss this warning, and then incorrectly believe your filters are configured as intended.

It doesn't look like httpd/config has a Validate method, but other configs do.

Reply:
Removed. It's now a native type for TOML so decoding will just fail.